code/modules/mob/living/silicon/pai/software.dm

```diff
@@ -619,5 +580,4 @@
 		internal_instrument = new(src)
 	var/dat = "<h3>Sound Synthesizer</h3>"
 	dat += "<a href='byond://?src=[REF(src)];software=loudness;sub=1'>Open Synthesizer Interface</a><br>"
-	dat += "<a href='byond://?src=[REF(src)];software=loudness;sub=2'>Choose Instrument Type</a>"
```

Comment:
Since there is only one option left in this menu, the "Sound Synthesizer" for the instrument should popup by default without having to click on it.

Reply:
Yeah, I already said in the PR body I would do it today, I just didn't have the time yesterday because of that internal remote signaler that took me way longer than I'd like to admit.

Reply:
Ahhh, I missed that.